Gems/AudioSystem/Code/Source/Editor/AudioControl.h

```diff
@@ -40,20 +40,20 @@ namespace AudioControls
         // will be pointing into the memory pool of an xml document that has gone out of scope.
         // This function is a rewritten version of 'clone_node' that does the deep copy of strings
         // into the new destination tree.
-        static AZ::rapidxml::xml_node<char>* DeepCopyNode(AZ::rapidxml::xml_node<char>* srcNode)
+        [[nodiscard]] static AZStd::unique_ptr<AZ::rapidxml::xml_node<char>> DeepCopyNode(AZ::rapidxml::xml_node<char>* srcNode)
         {
-            AZ::rapidxml::xml_node<char>* destNode = nullptr;
+            AZStd::unique_ptr<AZ::rapidxml::xml_node<char>> destNode{};
```

Comment:
The curly brace isn't needed here, the default constructor would initialize the unique_ptr member to `nullptr`